doc/administrator/administrate.rst

```diff
@@ -133,11 +133,17 @@ URL
 In the admin interface we can use in all the URL the following special schema:
 
 * ``static``: to use a static route,
-  * 'static:///icon.png' will get the URL of the 'static' static route of the project.
-  * 'static://c2cgeoportal/icon.png' will get the URL of the 'static' static route of 'c2cgeoportal'.
-  * 'static://prj:img/icon.png' will get the URL of the 'img' static route of 'prj'.
-
-* ``config``: to get the server name from the url,
-  if we use 'config://my_server/icon.png', with the config:
-  `servers: { "my_server": "http://example.com/test" }`
-  we will get the URL: 'http://example.com/test/icon.png'
+
+  * ``static:///icon.png`` will get the URL of the ``static`` static route of the project.
```

Comment:
The triple `/` is wanted, `static://c2cgeoportal/icon.png` is different than `static:///c2cgeoportal/icon.png`, the first point the file `/icon.png` in c2cgeoportal, the second `/c2cgeoportal/icon.png` in the project.